src/main/kotlin/domain/Cells.kt

```diff
@@ -1,20 +1,25 @@
 package domain
 
-class Cells(private val rows: List<Row>) {
-    fun getRows(): List<Row> = rows.toList()
+class Cells(private val initialRows: List<Row>) {
+    val rows: List<Row> get() = _rows
+    private val _rows: MutableList<Row> = initialRows.toMutableList()
```

Comment:
여기도 backing property를 잘 적용해 주셨어요. 일반적으로 backing property의 mutable은 상단에 위치하곤 합니다!
```suggestion
class Cells(private val initialRows: List<Row>) {
    private val _rows: MutableList<Row> = initialRows.toMutableList()
    val rows: List<Row> get() = _rows
```